alicloud/resource_alicloud_ots_table.go

```diff
@@ -0,0 +1,180 @@
+package alicloud
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/aliyun/aliyun-tablestore-go-sdk/tablestore"
+)
+
+func resourceAlicloudOtsTable() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAliyunOtsTableCreate,
+		Read: resourceAliyunOtsTableRead,
+		Update: resourceAliyunOtsTableUpdate,
+		Delete: resourceAliyunOtsTableDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"table_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"primary_key": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"type": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validatePrimaryTypeKey,
+						},
+					},
+				},
+				MaxItems: 4,
+			},
+			"table_option": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"time_to_live": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default: -1,
+						},
+						"max_version": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default: 1,
+						},
```

Comment:
Do the 'time_to_live' and 'max_version' have some limitation? If so, please specify them by validatefunc.